core/runtime/core_builtin.odin

```diff
@@ -591,7 +591,9 @@ assign_at_elems :: proc(array: ^$T/[dynamic]$E, index: int, args: ..E, loc := #c
 		copy(array[index:], args)
 		ok = true
 	} else {
-		resize(array, index+1+len(args), loc) or_return
+		new_size := index+len(args)
+		if len(array) == index && len(args) == 0 { new_size += 1 }
```

Comment:
```suggestion
```
This line causes an extra unwanted element, eg:
```
v: [dynamic]int
assign_at(&v, 0)
fmt.println(v) // [0] when it should still be empty/[].

v2: [dynamic]int
assign_at(&v2, 0, 1, 2, 3)
assign_at(&v2, 3)
fmt.println(v2) // [1, 2, 3, 0] when it should be [1, 2, 3].
```

Reply:
Soory I thought I should create a new position when entering a position that doesn't exist. But if that is not the case then i will except your chnages

Reply:
You may be right, lets see what Bill thinks because there aren't any docs/comments on this.

My intuition/reason was assigning nothing at the end should do nothing.
But I can also see wanting the assurance that `assign_at(&v, 4)` with any amount of args makes the array at least of length 4.